Makefile

```diff
@@ -152,9 +153,12 @@ clients/mockclients/cii_client.go: clients/cii_client.go | $(MOCKGEN)
 checks/mockclients/vulnerabilities.go: clients/vulnerabilities.go | $(MOCKGEN)
 	# Generating MockCIIClient
 	$(MOCKGEN) -source=clients/vulnerabilities.go -destination=clients/mockclients/vulnerabilities.go -package=mockrepo -copyright_file=clients/mockclients/license.txt
-cmd/packagemanager_mockclient.go: cmd/packagemanager_client.go | $(MOCKGEN)
+clients/packagemanager/packagemanager_mockclient.go: clients/packagemanager/client.go | $(MOCKGEN)
```

Comment:
I'm seeing changes if I force a re-generation. We should check in the auto generated version. 

```
touch clients/packagemanager/client.go
make generate-mocks
```